src/main/kotlin/com/terraformation/backend/search/SearchFilter.kt

```diff
@@ -116,7 +120,7 @@ data class FieldNode(
   }
 
   override fun toExactSearch(): FieldNode {
-    return if (type == SearchFilterType.Fuzzy) {
+    return if (type == SearchFilterType.Fuzzy && values.none { it == null }) {
```

Comment:
should this filter null values and use the non-empty result instead?

Reply:
i think i may have misread this line, will look at the updates. my comment doesn't seem right.